res/enkf/ert_template.py

```diff
@@ -43,9 +43,7 @@ def get_target_file(self):
     def get_args_as_string(self):
         """@rtype: str"""
         args_list = self._get_arg_list()
-        return ", ".join(
-            ["{}={}".format(key, args_list.get(key)) for key in args_list.keys()]
-        )
+        return ", ".join([f"{key}={args_list.get(key)}" for key in args_list.keys()])
```

Comment:
```suggestion
        return ", ".join([f"{key}={value}" for key, vlaue in args_list.items()])
```

Reply:
`args_list.items()` unfortunately not implemented.